### ISSUE TITLE
Fast tick rate disables volume control and settings menu (GameDAC)

### ISSUE THREAD

**Issue by LofhJann** [mtricht/gamesense-essentials#58]
When used with GameDAC, the way gamesense-essentials communicates with API disables normal functionality on OLED. This affects user experience negatively since on Arctis Pro Wireless all settings are edit using OLED menus. Also for some strange reason the volume control on GameDAC also is disabled with fast tick rate while on-headset volume works normally. 

Workaround is to set tick rate slower than default 50ms so there's time for menus to appear or disable clock and use menus while music is paused.

**Comment by mtricht** [comment]
@LofhJann I don't own either of these devices so I'm not sure what you're trying to accomplish. Inside Steelseries GG you should be able to chose what device the app uses, does that help?

**Comment by LofhJann** [comment]
Nah, the problem is with tick rate. LCD screens of SteelSeries when used with API don't care about other settings if API sends a message, API seems to always get priority over anything else. This causes strange behavior as stated previously. This is probably specific to Arctis Pro Wireless, since keyboards don't seem to rely on LCD for settings. Not sure if there is any real solution that could be applied since it seems to be API/App/Firmware issue, but something like adjusting tick/update rate if Arctis Pro is detected would be decent enough (not sure if this can be accomplished easily).
